cmd/validatemodels/validate_models.go

```diff
@@ -135,7 +135,13 @@ func ValidateAllAuthorizationModels(ctx context.Context, db storage.OpenFGADatas
 					}
 
 					if err != nil {
-						validationResult.Error = err.Error()
+						if invalidRelation, ok := err.(*typesystem.InvalidRelationError); ok {
+							// This will give a more detailed error
+							validationResult.Error = invalidRelation.Error() + " : " + invalidRelation.Cause.Error()
```

Comment:
i think it's simpler to remove this change and to change 

```go
func (e *InvalidRelationError) Error() string {
	return fmt.Sprintf("the definition of relation '%s' in object type '%s' is invalid", e.Relation, e.ObjectType)
}
```

to 


```go
func (e *InvalidRelationError) Error() string {
	return fmt.Sprintf("the definition of relation '%s' in object type '%s' is invalid: %s", e.Relation, e.ObjectType, e.Cause)
}
```

i  personally want the API to return the details, too